.github/pull_request_template.md

```diff
@@ -1,13 +1,20 @@
+<!-- Thank you for offering your contribution with Deals for Devs! -->
+
 ## Description
-<!-- Describe your contribution and its benefits in a sentence  -->
+<!-- Briefly explain what this PRs purpose is. -->
 
-## Related Ticket(s) and Documentation(s)
-<!-- Use `Closes #issue number` if your PR fixes an issue or `Relates #issue number` if it connects to the issue number -->
+## Issue
+<!-- If this PR is related to a specific issue # please list that here. (e.g - "Closes: XXXX") -->
 
-## Screenshots
+## Aditional Information
```

Comment:
Out of curiosity, wouldn't Additional Information allow for more open ended communication instead of only allowing screenshots in that area? Sometimes items may need clarification or points to be addressed? I'm fine with either, I just wanted to voice my opinion before making the change. :)

Reply:
Good question @BryanF1nes. In my experience with other projects, having a separate area for screenshots is best because for PRs where visual changes are crucial, maintainers and other contributors can have an easier time reviewing them.

Reply:
Good to know! Thanks I'll go ahead and make those changes! Thanks